bedrock/base/management/commands/get_geoip_db.py

```diff
@@ -0,0 +1,25 @@
+from django.conf import settings
+from django.core.management.base import BaseCommand, CommandError
+
+import requests
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        db_url = settings.MAXMIND_DB_URL
+        if not db_url:
+            return
+
+        try:
+            resp = requests.get(db_url, timeout=2, stream=True)
+        except requests.Timeout:
+            raise CommandError('Request timed out: ' + db_url)
+        except requests.RequestException as e:
+            raise CommandError(str(e))
+
+        if resp.status_code == 200:
+            with open(settings.MAXMIND_DB_PATH, 'wb') as geoip_file:
+                for chunk in resp:
+                    geoip_file.write(chunk)
+
+        self.stderr.write('GeoIP file downloaded successfully.')
```

Comment:
Why not `stdout` or event `logging`? Since this is not an error it shouldn't go to stderr.